actix-http/src/error.rs

```diff
@@ -304,6 +300,20 @@ impl From<httparse::Error> for ParseError {
     }
 }
 
+#[derive(Debug, Display)]
+/// A set of errors that can occur running blocking tasks in thread pool.
+pub enum BlockingError<E: fmt::Debug> {
+    #[display(fmt = "{:?}", _0)]
+    Error(E),
+    #[display(fmt = "Thread pool is gone")]
+    Canceled,
+}
+
+impl<E: fmt::Debug> std::error::Error for BlockingError<E> {}
```

Comment:
derive_more Error would delcare E the source

Reply:
This is copied from the actix-threadpool code and I believe it's explictly added by a PR.

Reply:
okay can do it later if needed